manager/src/lib.rs

```diff
@@ -616,7 +602,25 @@ impl CronManager {
 
         // only skip scheduling if user didnt intend
         let current_slot = self.get_slot_id(None);
-        let call_balance_used = self.task_balance_uses(&task);
+        // let call_balance_used = self.task_balance_uses(&task);
+        let call_balance_used = u128::from(env::used_gas());
+        let mut agent = self.agents.get(&env::signer_account_id()).unwrap();
```

Comment:
There may be some insane edge case where someone actually unregisters their agent.. Perhaps instead of `.unwrap()` an `.expect("message here")` 🤷🏼

Reply:
that would be an insane case for sure :P nice catch